standalone/Caddyfile

```diff
@@ -0,0 +1,3 @@
+localhost {
+        reverse_proxy helpdesk:8000
```

Comment:
```suggestion
        reverse_proxy django-helpdesk:8000
```
I had to change this to make it match the container name in the docker-compose file to make it work

Reply:
@timthelion Here I had to make it match the name of the container from the `docker-compose.yml`